golang/pkg/encryption/encryption.go

```diff
@@ -69,7 +69,7 @@ func GenerateEncryptedDocument(htmlStr, accessRequirement string, pubKeys map[st
 	if err = encryptAllSections(parsedHTML, encryptedSections, kh); err != nil {
 		return "", err
 	}
-	encryptedKeys, err := encryptDocumentKey(base64.StdEncoding.EncodeToString(ksEnc), accessRequirement, pubKeys)
+	encryptedKeys, err := encryptDocumentKey(base64.StdEncoding.EncodeToString(key.KeyValue), accessRequirement, pubKeys)
```

Comment:
Nit: I'd rather we pass `key.KeyValue` directly to `encryptDocumentKey` and apply base64 to it inside.

Reply:
Done.